docs/docs/quickstart.mdx

```diff
@@ -171,203 +143,260 @@ healthy system you're ready to roll! A healthy system will show something along
 the lines of (the order of messages might be reversed):
 
 ```
-kratos_1                      | time="2020-01-20T14:52:13Z" level=info msg="Starting the admin httpd on: 0.0.0.0:4434"
-kratos_1                      | time="2020-01-20T14:52:13Z" level=info msg="Starting the public httpd on: 0.0.0.0:4433"
+kratos_1 | time="2020-01-20T14:52:13Z" level=info msg="Starting the admin httpd on: 0.0.0.0:4434"
+kratos_1 | time="2020-01-20T14:52:13Z" level=info msg="Starting the public httpd on: 0.0.0.0:4433"
 ```
 
 :::note
 
 There are two important factors to get a fully functional system:
 
-- You need to make sure that ports `4455`, `4433`, `4434`, `4436` >
+- You need to make sure that ports `4455`, `4433`, `4434`, and `4436`&nbsp;
   [are free](https://serverfault.com/questions/309052/check-if-port-is-open-or-closed-on-a-linux-server).
-- Make sure to always use `127.0.0.1` as the hostname, never use `localhost`!
+- Make sure to always use `127.0.0.1` as the hostname; never use `localhost`!
   This is important because browsers treat these two as separate domains and
-  will therefore have issues with setting and using the right cookies.
+  will therefore have issues with setting and using cookies correctly.
 
 :::
 
-You might notice that no database is being used in this example. ORY Kratos
+
+
+You might notice that no database is being used in this example. Kratos
 supports SQLite, PostgreSQL, MySQL, and CockroachDB as database backends. For
 the quickstart, we're mounting a persistent volume to store the SQLite database
-in. Future guides will explain how to set up a production system!
+in.
 
-### Network Architecture
+Future guides will explain how to set up a production system.
 
-This demo makes use of several services / Docker Images:
+### Network architecture
+
+This demo makes use of several services:
 
 1. [ORY Kratos](https://github.com/ory/kratos)
-2. The **SecureApp** - an
-   [example application written in NodeJS](http://github.com/ory/kratos-selfservice-ui-node)
-   that implements the login, registration, logout, ..., and dashboard screen.
-3. An SMTP server with which ORY Kratos can send E-Mails with. We will use
-   [MailHog](https://github.com/mailhog/MailHog), a minimalistic SMTP throaway
-   server with an easy UI.
-
-To better understand how everything is wired, let's take a look at the network
+- Public / Browser API (port 4433)
+- Admin API (port 4434)
+2. [SecureApp](http://github.com/ory/kratos-selfservice-ui-node)
+- Public (port 4455) - an example application written in NodeJS
+   that implements the login, registration, logout, dashboard, and other UIs.
+3. [MailHog](https://github.com/mailhog/MailHog)
+- Public (port 4436) - a development SMTP server which Kratos will use to send emails.
+
+To better understand the application architecture, let's take a look at the network
 configuration. This assumes that you have at least some understanding of how
-Docker (Compose) Networks work:
+Docker networks work:
```

Comment:
```suggestion
Docker Networks work:
```

Reply:
I don't think Docker Network is a proper name, is it? Looking at [Docker's usage](https://docs.docker.com/network/), they refer to them as `Docker networks` because it's just a plain old network run by Docker, and not a brand name of some sort.

Reply:
Yup, my bad!